go-controller/pkg/ovn/namespace.go

```diff
@@ -94,6 +101,40 @@ func (oc *Controller) deletePodFromNamespaceAddressSet(ns string, ip net.IP) {
 	oc.setAddressSet(hashedAddressSet(ns), addresses)
 }
 
+// Creates an explicit "allow" policy for multicast traffic within the
+// namespace if multicast is enabled. Otherwise, removes the "allow" policy.
+// Traffic will be dropped by the default multicast deny ACL.
+func (oc *Controller) multicastAddNamespace(ns *kapi.Namespace) {
+
+	if !oc.multicastSupport {
+		return
+	}
+
+	enabled := (ns.Annotations[nsMulticastAnnotation] == "true")
+	enabledOld := oc.multicastEnabled[ns.Name]
+
+	if enabledOld == enabled {
+		return
+	}
+
+	if enabled {
```

Comment:
@dceara per below let's have this funciton log the errors. That would look liek:

```
var err error
if enabled {
	err = oc.createMulticastAllowPolicy(ns.Name)
} else {
	err = oc.deleteMulticastAllowPolicy(ns.Name)
}
if err != nil {
    logrus.Errorf(err.Error())
}
```

Reply:
Done.